content/en/tests/code_coverage.md

```diff
@@ -14,6 +14,10 @@ further_reading:
   text: "Learn about CI Monitors"
 ---
 
+{{< callout url="https://www.datadoghq.com/product-preview/code-coverage/" >}}
+Datadog is building a new dedicated Code Coverage product. Sign up for the Preview!
```

Comment:
```suggestion
Datadog Code Coverage is in Preview.
```

Reply:
This is non-blocking, you can take it or leave it. The button CTA is Request Access, but "Sign up for the Preview!" is peppier.

Reply:
@estherk15 and I chatted about this separately, summarizing: The preview signup is for a new, separate Code Coverage product, so just saying that "Code Coverage is in Preview" would make it sound like the current product is in Preview. I'll leave the wording as is to get the banner added, and we can revisit if needed.

Also we'll use the default button text for now; the headline and CTA in the body text convey some pep, too.